doc/projects_list.rst

```diff
@@ -0,0 +1,26 @@
+=======================
+Who uses Sphinx-Gallery
+=======================
+
+Here is a list of projects using `sphinx-gallery`.
+
+* `Sphinx-Gallery <https://sphinx-gallery.github.io/en/latest/auto_examples/index.html>`_
+* `Scikit-learn <http://scikit-learn.org/dev/auto_examples/index.html>`_
+* `Nilearn <https://nilearn.github.io/auto_examples/index.html>`_
+* `MNE-python <https://www.martinos.org/mne/stable/auto_examples/index.html>`_
+* `PyStruct <https://pystruct.github.io/auto_examples/index.html>`_
+* `GIMLi <http://www.pygimli.org/_examples_auto/index.html>`_
+* `Nestle <https://kbarbary.github.io/nestle/examples/index.html>`_
+* `pyRiemann <https://pythonhosted.org/pyriemann/auto_examples/index.html>`_
+* `scikit-image <http://scikit-image.org/docs/dev/auto_examples/>`_
+* `Astropy <http://docs.astropy.org/en/stable/generated/examples/index.html>`_
+* `SunPy <http://docs.sunpy.org/en/stable/generated/gallery/index.html>`_
+* `PySurfer <https://pysurfer.github.io/>`_
+* `Matplotlib <https://matplotlib.org/index.html>`_ `Examples <https://matplotlib.org/gallery/index.html>`_ and `Tutorials  <https://matplotlib.org/tutorials/index.html>`__
+* `PyTorch tutorials <http://pytorch.org/tutorials>`_
```

Comment:
Are we sure this is SG? I looked at it and it doesn't seem the same CSS, but maybe they modified it?

Reply:
It is SG : see https://github.com/pytorch/tutorials and https://github.com/pytorch/tutorials/blob/master/requirements.txt#L4 (too bad they pinned the version, though)